prow/plugins/lgtm/lgtm.go

```diff
@@ -30,9 +30,13 @@ import (
 	"k8s.io/test-infra/prow/repoowners"
 )
 
+// PluginName needs to be exported for checkconfig
 const PluginName = "lgtm"
 
 var (
+	addLGTMLabelNotification   = "LGTM label has been added.  <details>%s</details>"
```

Comment:
We should label the tree hash inside the details block so that users know what it is if they see it. e.g. `<details> Git tree hash: %s</details>`